tests/peft_integration/test_peft_models.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import inspect
+import unittest
+
+from transformers import AutoModelForCausalLM, OPTForCausalLM
+from transformers.testing_utils import require_peft, require_torch, require_torch_gpu, slow, torch_device
+from transformers.utils import is_peft_available, is_torch_available
+
+
+if is_peft_available():
+    from peft import PeftModel
+
+if is_torch_available():
+    import torch
+
+
+@require_peft
+@require_torch
+@slow
```

Comment:
Can we also add one non-slow test with a small fake model?

Reply:
Should be done, also added a new job to run peft tests thanks to @ydshieh